preload-webview.js

```diff
@@ -12,6 +12,10 @@ const clientInterface = client => {
     name: client.name,
     displayName: client.displayName,
     type: client.type,
+    api: client.api,
+    stdinWrite: payload => {
```

Comment:
should probably use `execute` api instead of this

Reply:
doesn't `execute` spawn a new binary process?

i need `stdinWrite` to write responses to clef, in accepting/rejecting requests. requests are passed in via the http rpc or IPC channel, and the secure response to them from the user needs to be passed in via stdin to the process. 

clef also has its own [internal api](https://github.com/ethereum/go-ethereum/tree/master/cmd/clef#internal-ui-api) that can only be accessed via stdin.